### ISSUE TITLE
Is campervan-hire.html still required as not used on site and old template

### ISSUE THREAD

**Issue by Conphilpott** [WamBamCampervan/htdocs#1]

**Comment by pegdigitaldesign** [comment]
Hi Connor, I've made some changes today just waiting to hear back from Martin.

I've not updated on server yet... Just in the stage3 folder.


I'll let you know once all is updated.


Nick


[1478689801392_pegdigitaldesign.png]


________________________________

**Comment by Conphilpott** [comment]
Ok, thanks for the update.

**Comment by pegdigitaldesign** [comment]
Hi Connor, can I update files on the server this week?


Regards


Nick



[1478689801392_pegdigitaldesign.png]


________________________________

**Comment by Conphilpott** [comment]
Yeah that’s fine with me, just let me know when you have completed please. 

Cheers,
Connor

**Comment by pegdigitaldesign** [comment]
Hi Connor, I'm updating files tomorrow. Can I update/overwrite files?

Or do you need them sent over to you/


Regards


Nick


[1478689801392_pegdigitaldesign.png]


________________________________

**Comment by Conphilpott** [comment]
Hi, 

Yeah it’s fine if you update them

**Comment by pegdigitaldesign** [comment]
OK thanks...



Nick


[1478689801392_pegdigitaldesign.png]


________________________________

**Comment by pegdigitaldesign** [comment]
Hi Connor, just made updated to site...


Cheers


Nick


[1478689801392_pegdigitaldesign.png]


________________________________

**Comment by Conphilpott** [comment]
ok thanks for the update

**Comment by Conphilpott** [comment]
i have added some tags to the html documents in the root directory of the site for google tag manager

**Comment by pegdigitaldesign** [comment]
Ok great stuff.
Thanks Connor
N

Peg Digital Design Ltd
pegdigitaldesign.co.uk<http://pegdigitaldesign.co.uk>

On 22 Dec 2017, at 22:08, Connor Philpott <notifications@github.com<mailto:notifications@github.com>> wrote:


i have added some tags to the html documents in the root directory of the site for google tag manager

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FWamBamCampervan%2Fhtdocs%2Fissues%2F1%23issuecomment-353682331&data=02%7C01%7C%7Cfb23bbe685024abbc17f08d54988911b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636495773248254663&sdata=oGPsoz5YfzLE7iuNKrslHXG2zn8R%2FqQcXgKzMCbdKFg%3D&reserved=0>, or mute the thread<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAglAlkoybxV48r0X4iG98_C0sp6Xz2J4ks5tDChfgaJpZM4QztyU&data=02%7C01%7C%7Cfb23bbe685024abbc17f08d54988911b%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636495773248254663&sdata=nX4gacTMeDfPhFNMTXnUkHKQXtYhWdV%2B%2BNpXdNszEZ4%3D&reserved=0>.

{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/WamBamCampervan/htdocs","title":"WamBamCampervan/htdocs","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/WamBamCampervan/htdocs"}},"updates":{"snippets":[{"icon":"PERSON","message":"@Conphilpott in #1: i have added some tags to the html documents in the root directory of the site for google tag manager"}],"action":{"name":"View Issue","url":"https://github.com/WamBamCampervan/htdocs/issues/1#issuecomment-353682331"}}}

**Comment by Conphilpott** [comment]
rich card added to index.html and marked up with comment

**Comment by Conphilpott** [comment]
initial seo setup complete, meta descriptions and titles changed on most pages, language defined on all pages etc..
